content/sensu-go/5.20/reference/backend.md

```diff
@@ -338,6 +340,23 @@ discovery instead of the static `--initial-cluster method`
 
 ### General configuration flags
 
+| annotations|      |
+-------------|------
+description  | Non-identifying metadata to include with event data that you can access with [event filters][9] and [tokens][27]. You can use annotations to add data that is meaningful to people or external tools that interact with Sensu.<br><br>In contrast to labels, you cannot use annotations in [API response filtering][25], [sensuctl response filtering][26], or [dashboard view filtering][54].
+required     | false
+type         | Map of key-value pairs. Keys and values can be any valid UTF-8 string.
+default      | `null`
+environment variable | `SENSU_ANNOTATIONS`
+example      | {{< highlight shell >}}# Command line examples
+sensu-backend start --annotations sensu.io/plugins/slack/config/webhook-url=https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
This example is fine as is, since a slack webhook in a handler asset is a valid use case.